apps/client/src/routes.js

```diff
@@ -53,62 +42,23 @@ export const routeConfig = [
       import(/* webpackPrefetch: true */ `./pages/IntroPage`)
     ),
   },
-  {
-    name: "location",
-    path: "/:slug/locatie",
-    component: React.lazy(() =>
-      import(/* webpackPrefetch: true */ `./pages/LocationPage`)
-    ),
-  },
-  {
-    name: "address",
-    path: "/:slug/adresgegevens",
-    component: React.lazy(() =>
-      import(/* webpackPrefetch: true */ `./pages/AddressPage`)
-    ),
-    matomoPage: "location-results",
-  },
-  {
-    name: "questions",
-    path: "/:slug/vragen/:question?",
-    component: React.lazy(() =>
-      import(/* webpackPrefetch: true */ `./pages/QuestionsPage`)
-    ),
-  },
-  {
-    name: "results",
-    path: "/:slug/uitkomsten",
-    component: React.lazy(() =>
-      import(/* webpackPrefetch: true */ `./pages/ResultsPage`)
-    ),
-  },
-  {
-    name: "conclusion",
-    path: "/:slug/conclusie",
-    component: React.lazy(() =>
-      import(/* webpackPrefetch: true */ `./pages/ConclusionPage`)
-    ),
-  },
   {
     name: "notfound",
     path: "*",
     component: React.lazy(() => import("./pages/NotFoundPage")),
   },
 ];
-
 export const redirectConfig = [
   [
     "/zonnepanelen-of-warmtecollectoren-plaatsen",
     "/zonnepanelen-of-zonneboiler-plaatsen",
   ],
 ];
-
 // build map of routes with `name` => `path`
 // ie. {intro: '/:slug/inleiding'}
 export const routes = Object.fromEntries(
   routeConfig.map(({ name, path }) => [name, path])
 );
-
 export const autofillRoutes = {
-  address: [routes.location, routes.address],
+  wrapper: [routes.wrapper],
 };
```

Comment:
`autofillRoutes` makes no sense in a one-pages.

Reply:
I agree, lets check for dataneeds on componentlevel in the refactor.